pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "fastapi-opa"
-version = "1.2.1"
+version = "2.3.1"
```

Comment:
It should be a minor increase, not a major, a minor and a bug fix simultaneously.

Reply:
Changed to 1.3.2

Reply:
Changed to 1.3.0